LiteCore/Support/Instrumentation.cc

```diff
@@ -20,28 +20,47 @@
 
 #ifdef __APPLE__
 #include <sys/kdebug_signpost.h>
+#include <os/signpost.h>
 #endif
 
 namespace litecore {
 
 #if defined(__APPLE__) && LITECORE_SIGNPOSTS
+
+    os_log_t LiteCore = os_log_create("com.couchbase.litecore", "signposts");;
```

Comment:
Should this be in `#if TARGET_OS_MACCATALYST` guard?

Reply:
yes. will update the PR